runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -234,13 +251,12 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
      * @param usernameFile   username file path
      * @param passwordFile   password file path
      */
-    public NettyClientRouter(String host, Integer port, Boolean tls,
+    public NettyClientRouter(NodeLocator node, Boolean tls,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)